### PR TITLE
Adding 'Upload' Function and sample test json

### PR DESCRIPTION
**Pull request by mtarng**
Unsure if we want to merge this as-is. This is the function code that's running on Azure Function currently. It still needs to be modified to call the integrating functions to the database. Do we also need to include template/deploy scripts on this?

### CODE REVIEW

**Comment by bnookala** [comment]
Check out the style and formatting samples for stuff here https://github.com/yokawasa/azure-functions-python-samples/tree/master/http-trigger-dump-request/function

autopep8 also

**Review by bnookala — CHANGES_REQUESTED** [review]
style and azure function definition json

**Comment by andrebriggs** [comment]
@mtarng any reason not to use the default __init__.py file name for the functions entry point?

File: functions/upload/run.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import os
+import json
+
+def get_id_from_filename(filename):
+    path_components = filename.split('/')
+    filename = path_components[-1]
+    return int(filename.split('.')[0])
+
+def process_vott_json(json):
```
Comment (andrebriggs) [inline functions/upload/run.py]: @mtarng eventually process_vott_json and get_id_from_filename() should be in a separate file. This way we aren't coupled to the the implementation of Azure Functions

Comment (andrebriggs) [inline functions/upload/run.py]: @mtarng eventually process_vott_json and get_id_from_filename() should be in a separate file. This way we aren't coupled to the the implementation of Azure Functions

**Comment by mtarng** [comment]
> @mtarng any reason not to use the default **init**.py file name for the functions entry point?

Just mimic'ing the default names generated by Azure. I'll change it up; learning Python as we go, please let me know standards/best practices as you see them 😄

**Review by andrebriggs — COMMENTED** [review]
> > @mtarng any reason not to use the default **init**.py file name for the functions entry point?
> 
> Just mimic'ing the default names generated by Azure. I'll change it up; learning Python as we go, please let me know standards/best practices as you see them 😄

See https://github.com/CatalystCode/active-learning-detect/blob/master/functions/pipeline/download/

**Review by andrebriggs — APPROVED** [review]

**Review by bnookala — APPROVED** [review]

**Pull request merged**